x/oracle/keeper/oracle.go

```diff
@@ -72,6 +72,28 @@ func (k Keeper) validateOracleRegistrationVote(ctx sdk.Context, signedVote *type
 	return nil
 }
 
+func (k Keeper) GetAllOracles(ctx sdk.Context) ([]types.Oracle, error) {
```

Comment:
Like this comments in [previous PR](https://github.com/medibloc/panacea-core/pull/364/files#r931841974), I'd like to discuss about naming of these kinds of functions too.

First of all, I prefer this naming when we get **all** items from KV store for `ExportGenesis`.

Why don't we name this kind of keeper function to `GetAll*`?

Likewise, we may need keeper functions to retrieve multiple items from KV store. In that case, I suggest to use `Get*List`. wdyt?

Reply:
Yes. It will be fine.
If so, proceed as follows.

`GetAllOracleList` -  gets all Oracles. (using ExportGenesis)
`GetOracleList` -  gets the paged Oracle list.

Reply:
It seems good to me!

Reply:
Thank you!
I fixed it.(https://github.com/medibloc/panacea-core/pull/366/commits/5f3e3f88a7a4423f46f0039485e37a3385404dd0)